parachain/primitives/core/src/outbound.rs

```diff
@@ -291,6 +291,14 @@ pub trait SendMessage {
 	fn deliver(ticket: Self::Ticket) -> Result<H256, SendError>;
 }
 
+/// A trait for getting the local costs associated with sending a message.
+pub trait SendMessageFee {
```

Comment:
Lets add a trait bound for `SendMessage`, since the outbound-queue pallet or any other implementation will usually implement both traits.

```
pub trait SendMessage: SendMessageFeeProvider
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/999/commits/5c03ad858d2f4c046b7dc26ce9cd6c1e5d615a34